presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -246,7 +246,7 @@ private Environment getEnvironment()
                     exposePort(container, 5007); // debug port
                 }
 
-                if (Boolean.getBoolean("CONTINUOUS_INTEGRATION")) {
```

Comment:
How about doing both system property and env var?

Reply:
But nobody passes the System property today? A system property already exists as `io.prestosql.testng.services.FlakyTestRetryAnalyzer.enabled` but we don't use it except in the tests of FlakyTestRetryAnalyzer.

Reply:
ok. let's leave it as it is then. thanks